7.2.0/oracle/org_data_fetch.sh

```
@@ -0,0 +1,80 @@
+  # Write processed line to the export file
+  echo "$TENANT_ID,$ORG_UUID" >> "$EXPORT_FILE"
+done < "$RAW_OUTPUT"
+
+# Clean up temporary raw output file
+rm -f "$RAW_OUTPUT"
+
+# Check if the export file contains more than just the header
+if [[ $(wc -l < "$EXPORT_FILE") -le 1 ]]; then
+  echo "Query executed successfully but no data returned. Exiting with success."
+  exit 0
+else
```

Shellcheck (0.11.0)

[error] Couldn't parse this else clause. Fix to allow more checks.

(SC1073)